packages/react-instantsearch-core/src/connectors/__tests__/connectInfiniteHits.js

```diff
@@ -21,7 +21,10 @@ describe('connectInfiniteHits', () => {
         results: { hits, page: 0, hitsPerPage: 2, nbPages: 3 },
       });
 
-      expect(props).toEqual({ hits, hasMore: true });
+      expect(props).toEqual({
+        hits: hits.map(hit => expect.objectContaining(hit)),
```

Comment:
same comment as for Hits, this could be better something more precise

Reply:
I added because now hits get `__position` and `__queryID`.
This tests is strictly about checking hits identity

Reply:
`hits.map(hit => expect.objectContaining(hit))` will always pass, except if hit isn't an object though. I don't see how this asserts anything.